airmozilla/starred/templates/starred/home.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'main/home.html' %}
+{% set page = page|default('starred') -%}
+
+{% block page_title %}
+Starred Events | {{ super() }}
+{% endblock %}
+
+{% block content_main %}
+<h2 class="section-title">Starred Events</h2>
+
+{% if events %}
+    {% include 'starred/events.html' %}
+{% else %}
+    <div id="events">
+        Please wait. Loading your starred events....
+    </div>
+{% endif %}
+
+{% endblock %}
+{% block site_js %}
+{{ super() }}
+    <script>
+    $(function() {
+    if (Stars.getStars().length) {
+            $('#events').load(window.location + '',
+                $.param({ids: Stars.getStars()}, true),
+                function () {
+                    Stars.triggerClickedStars();
+                }
+            );
+        } else {
+            $('#events').text("Sorry. You have no starred events");
```

Comment:
We need to fix this. First of all, "Sorry" isn't really right. It's nothing to be sorry about. :)

Secondly, I think we should avoid any English inside the Javascript (except some rare crazy error message that shouldn't happen e.g (`console.error("Seems your localStorage is broken. File a bug please");`)

What you should do is add a DOM element to the page. I'm thinking something like this:
```html
<div id="events">
    <p class="has-stars">Please wait. Loading your starred events...</p>
    <p class="no-stars" style="display:none">You have no starred events. Yet.</p>
</div>
```
Then, instead of using `$('#events').text(...)` we do something like 
```javascript
if (length) {
    $('#events').load(...);
} else {
    $('#events .has-stars').hide();
    $('#events .no-stars').show();  
}
```